subprojects/build-init/src/main/java/org/gradle/buildinit/plugins/internal/BuildScriptBuilder.java

```diff
@@ -103,76 +125,44 @@ public BuildScriptBuilder testRuntimeDependency(String comment, String... depend
         return dependency("testRuntime", comment, dependencies);
     }
 
-    /**
-     * Adds some arbitrary configuration to the _end_ of the build script.
-     */
-    public BuildScriptBuilder configuration(String comment, String config) {
-        this.config.add(new ConfigSpec(comment, Splitter.on("\n").splitToList(config)));
+    public BuildScriptBuilder taskMethodInvocation(String comment, String taskName, String taskType, String methodName) {
+        return configuration(
+            new TaskSelector(taskName, taskType),
+            new MethodInvocation(comment, methodName));
+    }
+
+    public BuildScriptBuilder taskPropertyAssignment(String comment, String taskName, String taskType, String propertyName, Object propertyValue) {
```

Comment:
:o: I noticed this method is only used by the test but I think that's ok in this case because:
 1. it's not a lot of code
 2. it sets an example for further enhancements